### PR TITLE
Release 0.8.2

### PR DESCRIPTION
**Pull request by mhadam**

### CODE REVIEW

**Comment by alexanderdean** [comment]
Removing myself as reviewer - @chuwy are you okay to take a look, or reassign if you would prefer someone else to review.

File: .travis/deploy.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 
 HOME = expanduser("~")
-DEFAULT_SERVER = 'https://pypi.python.org/pypi'
+DEFAULT_SERVER = 'https://upload.pypi.org/legacy/'
```
Comment (chuwy) [inline .travis/deploy.py]: I don't know what's the background of this change, but switching to something with `legacy` in URL looks slightly alarming to me.

Comment (mhadam) [reply .travis/deploy.py]: it's counter-intuitive but okay!
```(legacy in this URL refers to the fact that this is the new server implementation’s emulation of the legacy server implementation’s upload API.)```
https://packaging.python.org/guides/migrating-to-pypi-org/

Comment (alexanderdean) [reply .travis/deploy.py]: Nudge @mhadam , then ask Anton to re-review

Comment (mhadam) [reply .travis/deploy.py]: To fix this, I removed the DEFAULT_SERVER variable in `.travis/deploy.sh`.

**Comment by bernardosrulzon** [comment]
Guys, doesn't it make sense for this to go into this release as well? https://github.com/snowplow/snowplow-python-tracker/pull/188

**Comment by alexanderdean** [comment]
@mhadam @chuwy - what do you think?

**Comment by theosato** [comment]
The problem related and solved in #188 was already solved and merged in another PR? If not, what is the reason to don't merge?

**Pull request merged**